throttling/src/main/java/com/iluwatar/throttling/CallsCount.java

```diff
@@ -22,51 +22,56 @@
  */
 package com.iluwatar.throttling;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
 
 /**
  * A class to keep track of the counter of different Tenants
  * @author drastogi
  *
  */
 public final class CallsCount {
-  private static Map<String, Integer> tenantCallsCount = new ConcurrentHashMap<>();
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(CallsCount.class);
+  private static Map<String, AtomicLong> tenantCallsCount = new ConcurrentHashMap<>();
```

Comment:
Question: Why change from int to long? Why not `AtomicInteger`? Is it because the call count can breach MAXINT limit?

Reply:
Yes